### PR TITLE
Added passing of the getIsCustomRow and renderCustomRowContent callbacks into the BaseRow component.

### PR DESCRIPTION
**Pull request by dmitriyleonidovich**
The table supports passing the callbacks getIsCustomRow and renderCustomRowContent, but they are not forwarded to the BaseRow component.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "lint:prettier:fix": "prettier --write '**/*.{md,yaml,yml,json}'",
     "lint:styles": "stylelint 'src/**/*.scss' --report-needless-disables",
     "lint:styles:fix": "npm run lint:styles -- --fix",
-    "prepare": "husky",
+    "prepare": "npm run build",
```
Comment (beliarh) [inline package.json:35]: Why is this changed?

File: src/components/BaseTable/BaseTable.tsx [hunk 2]
```diff
@@ -142,6 +142,8 @@ export const BaseTable = React.forwardRef(
             stickyHeader = false,
             withFooter = false,
             withHeader = true,
+            getIsCustomRow,
+            renderCustomRowContent,
```
Comment (beliarh) [inline src/components/BaseTable/BaseTable.tsx:146]: Could you add a story in the BaseTable to show how it works?

**Review by beliarh — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (dmitriyleonidovich) [reply package.json:35]: Oh, this wasn't supposed to change; I made this modification in my own fork. It's been a while since I last worked with GitHub. I'll recreate the pull request from a separate branch.

Comment (beliarh) [reply package.json:35]: The changes moved here https://github.com/gravity-ui/table/pull/66/
